administrator-manual/en/accounts.rst

```diff
@@ -58,6 +58,12 @@ A group of user can be used to assign special permissions to some users or to cr
 
 As for the users, a group can be enabled to some (or all) services.
 
+.. tip:: For delegating permissions to the Server Manager, use the groups ``managers`` or ``administrators``.
+
+Two special groups exist, the users who belong in one of these groups are granted to the panels of the Server Manager
```

Comment:
"Two special groups exist...", this is not correct. The user must create these groups manually ...could you reword this line?